src/lib/chips/chip-list.ts

```diff
@@ -165,7 +165,18 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
   }
 
   /** The ARIA role applied to the chip list. */
-  get role(): string | null { return this.empty ? null : 'listbox'; }
+  @Input()
+  get role(): string | null {
```

Comment:
Note: I'm not a fan of this property signature (having `| null`), but I kept it for backwards compatibility. Since the property was readonly until now we could get away with changing it to just be a `string`.